pandas/tests/test_series.py

```diff
@@ -203,6 +203,28 @@ def f():
                 s.dt.hour[0] = 5
             self.assertRaises(com.SettingWithCopyError, f)
 
+    def test_strftime(self):
+        # GH 10086
+        s = Series(date_range('20130101', periods=5))
+        result = s.dt.strftime('%Y/%m/%d')
+        expected = Series(['2013/01/01', '2013/01/02', '2013/01/03', '2013/01/04', '2013/01/05'])
+        tm.assert_series_equal(result, expected)
+
```

Comment:
also test for periods, and a test for an invalid ``strftime`` formatter (e.g. the codes, I think raises ``ValueError``)